_docs/tasks/istio-auth.md

```diff
@@ -9,89 +9,100 @@ type: markdown
+* Completed steps 1 - 3 in [the Istio installation guide]({{home}}/docs/tasks/installing-istio.html#installing-on-an-existing-cluster).
 
-* Services in the same Kubernetes cluster but different namespaces are able to talk to each other through Istio Auth
-without extra trust setup.
+## Enabling Istio Auth
 
-#### Deplying CA
+### Deploying CA
 
-The following command creates  namespace *istio-system* and deploys CA into the namespace:
+The Istio CA only needs to be deployed once for the cluster.
+The following command creates namespace *istio-system* and deploys CA into the namespace:
 
 ```bash
-kubectl apply -f ./kubernetes/istio-auth/istio-cluster-ca.yaml
+kubectl apply -f templates/istio-auth/istio-cluster-ca.yaml
 ```
 
-#### <a name="istioconfig"></a>Enabling Istio Auth in Istio config
+### Deploying other services
 
-The following command uncomments the line *authPolicy: MUTUAL_TLS* in the file *kubernetes/istio-X.yaml*,
-and backs up the original file as *istio-X.yaml.bak*
-(*X* corresponds to the Kubernetes server version, choose "15" or "16").
+The following command will enable mTLS for the services in the "default" namespace,
+and the services are able to use the per-cluster CA deployed in the last step.
+Use the parameter *-n yournamespace* to specify a namespace other than the default one.
 
 ```bash
-sed "s/# authPolicy: MUTUAL_TLS/authPolicy: MUTUAL_TLS/" ./kubernetes/istio-X.yaml > ./kubernetes/istio-auth-X.yaml
+kubectl apply -f templates/istio-auth/isio-auth-per-cluster-ca.yaml
 ```
 
-#### Deploying other services
-
-Follow [the general Istio installation guide](./installing-istio.html),
-and **choose "If you would like to enable Istio Auth" in step 3**.
+Follow [the general Istio installation guide]({{home}}/docs/tasks/installing-istio.html) from step 5.
 
 ## Disabling Istio Auth
 
-Disabling Istio Auth requires all Istio services and applications to be reconfigured and restarted without auth config.
+This section shows how to disable Istio Auth in an Istio cluster.
+Disabling Istio Auth requires all Istio services and applications to be reconfigured and restarted with auth disabled.
+
+### Removing per-cluster Istio CA
+
+If you would like to disable auth for the entire cluster, you may want to remove the Istio CA.
+The following command removes Istio CA and its namespace *istio-system*.
+
+```bash
+kubectl delete -f templates/istio-auth/istio-cluster-ca.yaml
+```
 
-### For per-namespace CA Istio Auth
+### Redeploying Istio and applications
 
-Run the following command to uninstall Istio, and redeploy Istio without auth:
+Run the following command to uninstall auth-enabled Istio, and redeploy Istio without auth.
+Use the parameter *-n yournamespace* to specify a namespace other than the default one.
 
 ```bash
-kubectl delete -f ./kubernetes/istio-auth-X.yaml
-kubectl apply -f ./kubernetes/istio-X.yaml
+kubectl delete -f templates/istio-auth/istio-auth-per-cluster-ca.yaml
+kubectl apply -f istio.yaml
```

Comment:
You do not need to tell them to deploy istio.yaml.
Focus the task on setup and cleanup of auth.